bin/build-scxa-analytics-suggestions.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+#This below long curl request build suggestions on the solr server for the mentioned dictionaries as part of http request
+#Once build request completes, user can search for the suggestions
+# To search for the suggestions, here is the curl request:
+# curl -X GET 'http://localhost:8983/solr/scxa-analytics-v5/suggest?suggest=true&suggest.dictionary=ontologyAnnotationSuggester&suggest.dictionary=ontologyAnnotationAncestorSuggester&suggest.dictionary=ontologyAnnotationParentSuggester&suggest.dictionary=ontologyAnnotationSynonymSuggester&suggest.dictionary=ontologyAnnotationChildSuggester&suggest.q=skin'
+
+curl -X GET 'http://localhost:8983/solr/scxa-analytics-v5/suggest?suggest=true&suggest.build=true&suggest.dictionary=ontologyAnnotationSuggester&suggest.dictionary=ontologyAnnotationAncestorSuggester&suggest.dictionary=ontologyAnnotationParentSuggester&suggest.dictionary=ontologyAnnotationSynonymSuggester&suggest.dictionary=ontologyAnnotationChildSuggester'
```

Comment:
Use the collection and its version from the set environment variables instead of hard coding it in the URL please. As it is done here for instance: https://github.com/ebi-gene-expression-group/index-scxa/blob/develop/bin/create-scxa-gene2experiment-collection.sh#L8. Same for the host, as shown on the example.

Reply:
Please also add a line on the README about this, and a call on the bats tests to use this command.

Reply:
Thanks for the review @pcm32, Sure I will fix them.